Framework/Beamline/test/ComponentInfoTest.h

```diff
@@ -755,7 +755,7 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     TS_ASSERT(compInfo.relativePosition(subComponentIndex)
                   .isApprox(subCompPosition - rootPosition));
 
-    const auto diffPos =
+    const Vector3d diffPos =
```

Comment:
Could we add a comment explaining why `auto` is not a good idea, to save this bug creeping back in?

Reply:
:+1: